support/rollup/factory.js

```diff
@@ -0,0 +1,99 @@
+import path from 'path'
+import { startCase } from 'lodash'
+import alias from 'rollup-plugin-alias'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+
+export default (pkg) => {
+  // UMD build for browsers
+  const umdConfig = {
+    input: `packages/${pkg.name}/src/index.js`,
+    output: {
+      file: `packages/${pkg.name}/${pkg.browser}`,
```

Comment:
Same concept here, can you store all of the `pkg` props as interim variables, so that we don't need to keep repeating `pkg.`?